File: python/07_maps.py
# 07 dictionaries / maps

person = {
    "name": "ulas",
    "age": 20,
    "city": "istanbul"
}

print(person["name"])
print(person.get("age"))

person["age"] = 21
person["job"] = "student"
print(person)

for key, value in person.items():
    print(key, "->", value)

File: php/07_maps.php
<?php
// 07 maps / associative arrays

$person = [
    "name" => "ulas",
    "age" => 20,
    "city" => "istanbul"
];

echo $person["name"] . "\n";
echo $person["age"] . "\n";

$person["age"] = 21;
$person["job"] = "student";

print_r($person);

foreach ($person as $key => $value) {
    echo $key . " -> " . $value . "\n";
}
